libcxx/include/cmath

```diff
@@ -312,33 +312,19 @@ long double    truncl(long double x);
 #pragma GCC system_header
 #endif
 
+#ifdef _LIBCUDACXX_COMPILER_NVRTC
+#include <math_constants.h>
```

Comment:
Just FYI: I am not sure what's the release strategy of libcu++, so maybe this can be ignored, but including this header has caused trouble in CuPy when a user runs CUDA + NVRTC + CuPy in a "runtime" environment (such as the CUDA Runtime Docker, or RAPIDS's favorite Conda env), because none of the standard CUDA headers are present there due to EULA.

Reply:
Thanks Leo!

That's insightful, I'm recalling now similar issues with cooperative_groups being unavailable in NVRTC. 

I have a more preferable solution now that I think about it.

Reply:
Wesley, the change in 6696f20 is cool 😄